cypress/integration/registration.spec.js

```diff
@@ -18,8 +18,8 @@ describe('registration', () => {
   })
 
   after(() => {
-    authModule.logout(),
-    cy.url().should('eq', 'https://cypress.vivifyscrum-stage.com/login')
+    cy.logout(),
+    cy.url().should('eq', `${Cypress.config('baseUrl')}/login`)
```

Comment:
Since you repeat this all over the place, could it become a command then?

```suggestion
    Cypress.Commands.add('assertUrl', (url) => {
        cy.url().should('eq', url)
    });
```

You would, of course, put this into an appropriate `commands.js` file or similar.

Usage: 

```javascript
cy.assertUrl(`${Cypress.config('baseUrl')}/login`);
```